<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QWidget" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chat</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="messageBrower"/>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QFontComboBox" name="fontComboBox"/>
            </item>
            <item>
             <widget class="QComboBox" name="sizeComboBox">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>11</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>12</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>13</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>14</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>15</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>17</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>18</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>19</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>21</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="boldToolBtn">
              <property name="toolTip">
               <string>字体</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="pic.qrc">
                <normaloff>:/C:/Users/asus/Pictures/rc/chat/bold.png</normaloff>:/C:/Users/asus/Pictures/rc/chat/bold.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="italicToolBtn">
              <property name="toolTip">
               <string>斜体</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="pic.qrc">
                <normaloff>:/C:/Users/asus/Pictures/rc/chat/leek.png</normaloff>:/C:/Users/asus/Pictures/rc/chat/leek.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="underlineToolBtn">
              <property name="toolTip">
               <string>下划线</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="pic.qrc">
                <normaloff>:/C:/Users/asus/Pictures/rc/chat/xiahuaxian.png</normaloff>:/C:/Users/asus/Pictures/rc/chat/xiahuaxian.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="colorToolBtn">
              <property name="toolTip">
               <string>颜色</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="pic.qrc">
                <normaloff>:/C:/Users/asus/Pictures/rc/chat/about.png</normaloff>:/C:/Users/asus/Pictures/rc/chat/about.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="saveToolBtn">
              <property name="toolTip">
               <string>发送信息</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="pic.qrc">
                <normaloff>:/C:/Users/asus/Pictures/rc/chat/send.png</normaloff>:/C:/Users/asus/Pictures/rc/chat/send.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="sendToolBtn">
              <property name="toolTip">
               <string>保存</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="pic.qrc">
                <normaloff>:/C:/Users/asus/Pictures/rc/chat/file.png</normaloff>:/C:/Users/asus/Pictures/rc/chat/file.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="clearToolBtn">
              <property name="toolTip">
               <string>清理</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="pic.qrc">
                <normaloff>:/C:/Users/asus/Pictures/rc/chat/clear.png</normaloff>:/C:/Users/asus/Pictures/rc/chat/clear.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="messageTextEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>120</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="userTableWidget">
        <column>
         <property name="text">
          <string>用户名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>主机</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>地址</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>77</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="sendPushButton">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>77</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="userNumLabel">
        <property name="text">
         <string>在线人数:                           </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>77</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitPushButton">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
